Documentation/MigrateToComposer/MigrationSteps.rst

```diff
@@ -13,14 +13,30 @@ Migration steps
     *   `Basic usage of Composer <https://getcomposer.org/doc/01-basic-usage.md>`__
 
 
+It is recommended to perform a Composer migration using the latest TYPO3 major release to prevent
+facing bugs and issues that have been solved already in newer versions. In case you
+are using an older TYPO3 version in Legacy installation mode, you have two viable options:
+
+*   **Upgrade TYPO3 Legacy first**, then do the migration to Composer. This is probably
+    easiest, as you can follow the :ref:`Legacy Upgrade Guide <legacy>`, and then this guide.
```

Comment:
I suggest to avoid "easiest". We want to get rid of "easy", "simple" etc, as it is mostly not easy. And upgrading a legacy installation is mostly not easy.

Reply:
Well, between the two ways this is certainly "easiest", because the other way involves a TYPO3 major update.

I understand your intention. I rephrased it to "straight-forward". I don't think one way or the other can really be recommended here. I personally would perform the upgrade in one go, but I removed that due to the feedback from you. Which route which you recommend? How would you rephrase it?